### PR TITLE
Add copy button for markdown code blocks

### PR DESCRIPTION
**Pull request by pfeuffer**
## Proposed changes

Adds a small "copy to clipboard" button on the top
right edge of code blocks in markdown content.

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] PR is well described and the description can be used as commit message on squash
- [x] Target branch is not master (in most cases develop should bet the target of choice) 
- [x] Code does not conflict with target branch
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog`
- [x] Definition of Done's fulfilled: [DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/definition-of-done.md) // [UI DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/ui-dod.md)

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: scm-ui/ui-components/src/SyntaxHighlighter.tsx [hunk 1]
```diff
@@ -56,6 +69,11 @@ const SyntaxHighlighter: FC<Props> = ({ language = defaultLanguage, showLineNumb
     value
   );
 
+  const copy = () => {
+    navigator.clipboard.writeText(value);
+    setCopied(true);
```
Comment (eheimbuch) [inline scm-ui/ui-components/src/SyntaxHighlighter.tsx:74]:
```suggestion
copyToClipboard(value).then(() => setCopied(true));
```

Why not like this? We have an exported copyToClipBoard function. Just for consistence

File: scm-ui/ui-components/src/SyntaxHighlighter.tsx [hunk 2]
```diff
@@ -74,7 +92,10 @@ const SyntaxHighlighter: FC<Props> = ({ language = defaultLanguage, showLineNumb
       >
         {value}
       </ReactSyntaxHighlighter>
-    </div>
+      <TopRightButton title={t("syntaxHighlighting.copyButton")} onClick={copy}>
+        <i className={copied ? "fa fa-clipboard-check" : "fa fa-clipboard"} />
+      </TopRightButton>
```
Comment (eheimbuch) [inline scm-ui/ui-components/src/SyntaxHighlighter.tsx:97]: This new button appears underneath the "markdown-switch-button" and is nearly invisible. This is not what you intended, is it?

**Review by eheimbuch — CHANGES_REQUESTED** [review]

File: scm-ui/ui-components/src/SyntaxHighlighter.tsx (hunk 1 above)
Comment (pfeuffer) [reply scm-ui/ui-components/src/SyntaxHighlighter.tsx]: Didn't know about this one, thx!

File: scm-ui/ui-components/src/SyntaxHighlighter.tsx (hunk 2 above)
Comment (pfeuffer) [reply scm-ui/ui-components/src/SyntaxHighlighter.tsx:97]: Ah, ok, this is ... hmpf. Do you have a suggestion?

**Pull request closed without merging**